### ISSUE TITLE
Using node-nightly behind a proxy

### ISSUE THREAD

**Issue by mikaturk** [hemanth/node-nightly#35]
Hey,

I want to use node-nightly at school but we have a proxy, i've already configured git and npm to use the proxy, but when i type `node-nightly` it says i have no internet connection

Thanks,
Mika

**Comment by hemanth** [comment]
Check #34

**Comment by mikaturk** [comment]
@hemanth no I meant that I'd like the download to work behind a proxy

**Comment by hemanth** [comment]
Yup, can you log the `URL` and access it directly and see what does it result in?  

Proxy shouldn't matter, unless the URL is blocked by the proxy.

**Comment by mikaturk** [comment]
@hemanth I can download it just fine, not on my laptop ATM, but it worked on my android phone which is behind the same proxy
https://goo.gl/photos/7NAnvuSZNbuSdSQL7
